src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/SeriesWrapper.tsx

```diff
@@ -92,6 +92,10 @@ const SubSeriesHeaderText = styled(Box)`
   left: 0;
   top: 50px;
   right: 0;
+  mix-blend-mode: difference;
+  color: ${color("white100")};
+  background: ${color("black100")};
+  z-index: -1;
```

Comment:
Literally the only feedback I can come up with here is that `z-index` and `mix-blend-mode` are swapped in this file relative to the last file. 🥇

Reply:
Yeah, I should alphabetize all the properties, good point!